profile/README.md

```diff
@@ -14,9 +14,19 @@ There is currently an implementation effort working on creating the first versio
 <!--
 - A "Getting involved" section that explains which public, archived, linkable communication channels the project uses. This should include a link to the project issue tracker, but also to any further discussion media used.
 -->
+If you want to get involved in the project with us, you can take a look at the [Registry project's Readme](https://github.com/project-origin/registry#readme), or at the [Partner Collaboration Readme](https://github.com/project-origin/origin-collaboration#readme) for an overview of what we are doing right now. 
```

Comment:
Yup, this is solid. The partner collab repo will 404 for the general public as they don't have the access rights. 
We can just leave it at that now, but maybe add some extra 3 words along the lines of expectation management "this is 404 if you're not part of this or that group, see here for becoming partner"